### DISCUSSION TITLE
Reduce tracer biases in the Black Sea

### DISCUSSION THREAD

**Discussion by gustavo-marques** [NCAR/omwg_dev#36] [Biases]
The purpuse of simulations [024](https://github.com/NCAR/omwg_dev/issues/32) and [025](https://github.com/NCAR/omwg_dev/issues/33) is to check if  narrowing and shallowing the Bosphorus and Dardanelles straights reduces the tracer biases in the Black Sea. These runs should be compared against [022](https://github.com/NCAR/omwg_dev/issues/24) and [023](https://github.com/NCAR/omwg_dev/issues/26), respectively. 

 Below are temperature and salinity drifts from the WOA18 annual mean for the Black Sea. The biases are signfincatly reduced in 024 and 025. Interstingly, the near-surface salinity bias flipped sign in the new runs. I think we should incorporate the changes made in these runs into our next baselines. 

### Temperature bias in the Black Sea

![image](https://github.com/NCAR/omwg_dev/assets/11339137/a4eeff02-a1f0-4daa-b5fd-581e4c12a4df)
![image](https://github.com/NCAR/omwg_dev/assets/11339137/6c4cad06-6c41-4c47-a720-0b00a34eab58)
![image](https://github.com/NCAR/omwg_dev/assets/11339137/3b54615b-47b5-400d-9543-5456080af4e3)
![image](https://github.com/NCAR/omwg_dev/assets/11339137/e5921de6-9b9e-4432-836e-0a0a7367c6a3)

### Salinity bias in the Black Sea
![image](https://github.com/NCAR/omwg_dev/assets/11339137/b6d8f770-11fd-436c-85d2-e8741e0c265c)
![image](https://github.com/NCAR/omwg_dev/assets/11339137/a6a76784-45ab-4a98-864d-90a656c6386c)
![image](https://github.com/NCAR/omwg_dev/assets/11339137/82a8f265-5e05-4494-bd67-027df0d408bf)
![image](https://github.com/NCAR/omwg_dev/assets/11339137/0647271e-01c3-48cc-8c28-8c0ffcbe8380)

cc'ing @klindsay28, @fobryan3

**Comment by klindsay28** [comment]
Here are some line plots of 1000m averages over the Black Sea for tracers:
![image](https://github.com/NCAR/omwg_dev/assets/15570875/4cdc685c-b9c3-47f5-8d89-343fadb8b05f)
![image](https://github.com/NCAR/omwg_dev/assets/15570875/4f878a3b-8be6-46ed-9b13-0242b0574fa5)
![image](https://github.com/NCAR/omwg_dev/assets/15570875/15cc8b08-bb52-478d-8360-ff840255ccb7)
The topography and width edits are successfully removing the spurious deep trends in `thetao` and `so`.
Additionally, `agessc` over this initial segment appears to be equal to `time`, while in the control runs it had a smaller trend, indicative of younger Med water being transported in to the Black Sea.
These are all in the desired direction.

Here are some plots that I don't quite understand when taken together:
![image](https://github.com/NCAR/omwg_dev/assets/15570875/4198af70-984c-4265-bb65-36139b08deee)
![image](https://github.com/NCAR/omwg_dev/assets/15570875/baeb9b50-aaff-4054-88d7-8739971e9b6f)
![image](https://github.com/NCAR/omwg_dev/assets/15570875/b78ccc3b-79f4-49cf-bdf2-260992386b1c)
So `sst` is slightly up and `sos` is down with the mods, but `mlotst` and `oml (not shown) are up.
I'm not sure why the mods lead to deeper mixing. I'm not particularly concerned by this, but do find it curious.
